jaggery-modules/utils/module/scripts/request/request.js

```diff
@@ -68,4 +68,24 @@ var log = new Log('request_module');
         });
         return obj;
     };
+    /*
+     * Format the encoded  query string and return user friendly search query
+     * */
+    request.formatSearchQuery = function (queryString) {
+        var searchQuery = "";
+        var qjson = parse('{' + queryString + '}');
+        var searchKeys = Object.keys(qjson);
+        if ((searchKeys.length === 1) && (searchKeys.indexOf("name") >= 0)) {
+            searchQuery += qjson[searchKeys.pop()];
+        }
```

Comment:
Thanks @Kishanthan , This has been fixed in [this commit](https://github.com/wso2/carbon-store/commit/c85def0742c5654f1efc691e7db53f865d3d89de)